Gemfile

```diff
@@ -33,6 +33,7 @@ gem 'sprockets-rails', '>= 2.3.3', '< 3'
 gem 'responders', '~> 2.0'
 gem 'roadie-rails', '~> 1.1'
 gem 'x-editable-rails', '~> 1.5.5'
+gem 'webpack-rails', '~> 0.9.7'
```

Comment:
https://github.com/mipearson/webpack-rails/commit/2fb0e85c20aa97819e9850e61261d98f777301b6
it looks like 0.9.8 is enough